com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/cache/ResourceCacheKey.java

```diff
@@ -28,6 +28,32 @@
 
 public class ResourceCacheKey {
 
+	/*
+	 * This list of options have no impact on the created resources
+	 * and can safely be ignored when calculating the resource key
+	 */
+	private static Set<String> IGNORED_OPTIONS = Set.<String>of(
+		"verbose",
+		"archive",
+		"email",
+		"auth",
+		"bundle-output",
+		"bundle-format",
+		"mobileprovisioning",
+		"identity",
+		"keystore",
+		"keystore-pass",
+		"keystore-alias",
+		"debug",
+		"debug-ne-upload",
+		"variant",
+		"strip-executable",
+		"with-symbols",
+		"build-report",
+		"build-report-html",
+		"architectures"
+	);
```

Comment:
These don't currently affect the content tasks iirc. But in the future, they might?
We're currently producing the merged manifest file, and other resources

* build-server
* defoldsdk
* build-artifacts

What about these?
* resource-cache-local
* resource-cache-remote
* resource-cache-remote-user
* resource-cache-remote-pass
* binary-output

Reply:
I wonder if it's too easy to forget this list when adding options.
Could this be somehow more connected to where we define the properties in Bob.java?

Reply:
Good point. I've changed so that when we define the options in Bob.java we also set if the option impacts the cache or not.